mod/format/assets/script/decko/slotter.js.coffee

```diff
@@ -201,8 +203,8 @@ jQuery.fn.extend
     slotter.addClass "_close-#{type}"
 
   slotterBeforeSend: (opt) ->
-    return if opt.noSlotParams
-
     # avoiding duplication. could be better test?
-    unless (opt.url.match(/home_view/) or @data("slotter-mode") == "modal")
-      opt.url = decko.slot.path opt.url, @slot()
+    return if opt.noSlotParams or opt.url.match(/home_view/) or
+      @data("slotter-mode") == "modal" or @data("slotter-mode") == "override"
```

Comment:
this is a little cognitively demanding.

Reply:
I thought three "or"s  are less cognitively demanding than a mixture of "if" and "unless"

Reply:
neither are very good though, right?  might benefit from its own function.